cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java

```diff
@@ -66,6 +68,13 @@ public Cocos2dxWebView(Context context, int viewTag) {
 
         this.setWebViewClient(new Cocos2dxWebViewClient());
         this.setWebChromeClient(new WebChromeClient());
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```

Comment:
`setWebContentsDebuggingEnabled` has been added in Android's API level 19 (KITKAT). There is no sense to call it on platforms with API version < 19. 

As you can see in the [build log](http://45.56.80.45:8080/job/ccs-pr/node=android/4351/console), there are two errors:
```
[javac] Compiling 28 source files to /android_slave/workspace/ccs-pr/node/android/cocos/platform/android/java/bin/classes
    [javac] /android_slave/workspace/ccs-pr/node/android/cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java:71: error: cannot find symbol
    [javac]         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
    [javac]                                                         ^
    [javac]   symbol:   variable KITKAT
    [javac]   location: class VERSION_CODES
    [javac] /android_slave/workspace/ccs-pr/node/android/cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java:75: error: cannot find symbol
    [javac]                 WebView.setWebContentsDebuggingEnabled(true);
    [javac]                        ^
    [javac]   symbol:   method setWebContentsDebuggingEnabled(boolean)
    [javac]   location: class WebView
    [javac] Note: /android_slave/workspace/ccs-pr/node/android/cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxDownloader.java uses unchecked or unsafe operations.
    [javac] Note: Recompile with -Xlint:unchecked for details.
    [javac] 2 errors
```
Both `Build.VERSION_CODES.KITKAT` and `setWebContentsDebuggingEnabled(boolean)` don't exist in the version of target API. In the [build log](http://45.56.80.45:8080/job/ccs-pr/node=android/4351/console) we can find that Ant is using API 10 (Android 2.3.3). 
```
-build-setup:
[getbuildtools] Using latest Build Tools: 23.0.0 rc2
     [echo] Resolving Build Target for cocos2dxandroid...
[gettarget] Project Target:   Android 2.3.3
[gettarget] API level:        10
     [echo] ----------
     [echo] Creating output directories if needed...
    [mkdir] Created dir: /android_slave/workspace/ccs-pr/node/android/cocos/platform/android/java/bin
    [mkdir] Created dir: /android_slave/workspace/ccs-pr/node/android/cocos/platform/android/java/bin/res
    [mkdir] Created dir: /android_slave/workspace/ccs-pr/node/android/cocos/platform/android/java/bin/rsObj
    [mkdir] Created dir: /android_slave/workspace/ccs-pr/node/android/cocos/platform/android/java/bin/rsLibs
    [mkdir] Created dir: /android_slave/workspace/ccs-pr/node/android/cocos/platform/android/java/gen
    [mkdir] Created dir: /android_slave/workspace/ccs-pr/node/android/cocos/platform/android/java/bin/classes
    [mkdir] Created dir: /android_slave/workspace/ccs-pr/node/android/cocos/platform/android/java/bin/dexedLibs
     [echo] ----------
     [echo] Resolving Dependencies for cocos2dxandroid...
[dependency] Library dependencies:
[dependency] No Libraries
[dependency] 
[dependency] ------------------
[dependency] API<=15: Adding annotations.jar to the classpath.
```
I checked the configuration of that test (CppTest). It is in `tests/cpp-tests/proj.android/project.properties` and 
`cocos/platform/android/java/project.properties`. `target` should be set there to `android-22`
If you run test using gradle everything should be fine.

In conclusion, I suggest updating `targetSdkVersion` of the app to use the same SDK in proj.android and proj.android-studio.